### PR TITLE
Fix style in options pages

### PR DESCRIPTION
**Pull request by IgorA100**

### CODE REVIEW

File: web/skins/classic/css/dark/skin.css [hunk 1]
```diff
@@ -42,6 +42,10 @@ h4 {
     color: #117AAd;
 }
 
+a {
```
Comment (connortechnology) [inline web/skins/classic/css/dark/skin.css]: This shouldn't be neccessary as you've added it to the base... skin.css.. or base/skin.css.  We always include base.

Comment (IgorA100) [reply web/skins/classic/css/dark/skin.css]: Of course you're right, sorry.
I changed code.

**Comment by connortechnology** [comment]
This is all fine... but in general we have an inconsistency about where to put the buttons, top or bottom.  I was leaning towards top, and generally having them stay sticky so that they are always available to click.  We can address that in another PR though. Just wanted to bring it up.

**Comment by IgorA100** [comment]
Yes, I should have asked about the button placement. But, on this layout there is no problem moving them up. 



Most of the buttons for working with tables were at the bottom, so I moved the rest down.
The "Save" button is still at the top.
With buttons sticking when scrolling, it is necessary to leave the header motionless.
Perhaps we should think about a new accordion-style left navigation menu design and reconsider the top menu in the header?

**Comment by IgorA100** [comment]
If scroll table with thead fixed, then at high resolution everything will be beautiful, but at low resolution everything will be bad from the fixed filters.
Land01
Land1

**Comment by connortechnology** [comment]
Console needs a lot of work. Filters should not all be visible but show on demand. And collapse.

**Pull request merged**